Apps/WebClient/src/ClientApp/src/components/calendar/header.vue

```diff
@@ -64,6 +64,7 @@
         </b-col>
         <b-col class="header-right">
             <slot name="header-right"> </slot>
```

Comment:
well what do you know, looks like there was already a slot for that! Lets remove the header-right or rename the month-list-toggle slot to match this

Reply:
Removing header-right